src/style.css

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+}
+
+/* @font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+} */
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf") format("TrueType");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc") format("TrueType");
+  font-weight: 600;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf") format("TrueType");
+  font-weight: 900;
+}
+
+.search {
+  margin-top: 20px;
+  font-family: Avenir, Heavy, sans-serif;
+}
+
+.search-bar {
+  width: 100%;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  font-family: inherit;
+}
+
+.search-bar::placeholder {
+  color: #3d4e61;
+}
+
+/* ::placeholder {
+  color: #3d4e61;
+} */
+
+.big-input {
+  height: 70px;
+
+  background-position: 26px 25px;
+  background-size: 19;
+  
+  padding-left: 62px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+
+  font-weight: 300px;
+  font-size: 16px;
+}
+
+.big-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.big-input:focus {
+  background: linear-gradient(180deg, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.small-input {
```

Comment:
check this checklist point
no need to name it as big and small

just use this example
<img width="935" alt="image" src="https://user-images.githubusercontent.com/102411130/233300497-b1eaa7be-30e2-459b-beb1-582367142dcf.png">